src/myst.ts

```diff
@@ -104,7 +105,7 @@ export class MyST {
 
   _createTokenizer() {
     const exts = this.opts.extensions;
-    const tokenizer = MarkdownIt('commonmark', this.opts.markdownit);
+    const tokenizer = MarkdownIt(MARKDOWN_IT_CONFIG as any, this.opts.markdownit);
```

Comment:
The `MarkdownIt` type requires a `presetName` string, but passing a config JSON works. This is only slightly risky when we bump `markdown-it` version.